### PR TITLE
PostCSS plugin for @darktheme directive

### PR DESCRIPTION
**Pull request by fallaciousreasoning**
Hey Pete, what do you think of this? I had a bit of trouble debugging and getting it to work nicely with Svelte, but I think it's working now.

### CODE REVIEW

File: postcss/darkmode.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        console.log(rule.toString())
+        return ''
+    }
+    const selectors = rule.selector.split(',').map(s => s.trim()).map(transform)
+    return `${selectors.join(', ')} {
+${indent(rule.nodes.map(n => n.toString() + ';').join('\n'), 1)}
+}`
+}
+
+module.exports = (options = { forceGlobal: true }) => {
```
Comment (AlanBreck) [inline postcss/darkmode.js:16]: Can we actually default this to false? It seems the most intuitive would be to assume it won't, and then change that flag explicitly in the svelte config.

**Review by AlanBreck — COMMENTED** [review]
This is awesome!

File: postcss/darkmode.js (hunk 1 above)
Comment (fallaciousreasoning) [reply postcss/darkmode.js:16]: Yup, I went back and forwards on this a few times but happy to default it to false :smile:

**Comment by AlanBreck** [comment]
Should we close this since you've moved to a different repo?

**Comment by fallaciousreasoning** [comment]
Yeah, probably a good call. Closing (see https://github.com/fallaciousreasoning/postcss-plugin-darkmode)

**Pull request closed without merging**